package-examples/hello-composite-pkg/README.md

```diff
@@ -0,0 +1,106 @@
+  Get the example composite package on to local using `kpt pkg get`
+  
+  $ kpt pkg get https://github.com/GoogleContainerTools/kpt.git/package-examples/hello-composite-pkg \
+    hello-composite-pkg
+  
+      fetching package /package-examples/hello-composite-pkg from https://github.com/GoogleContainerTools/kpt to hello-composite-pkg
+  
+  ### View the package contents
+  List the package contents in a tree structure.
+  
+  $ kpt cfg tree hello-composite-pkg/
+  
+      hello-composite-pkg
+      ├── [Kptfile]  Kptfile hello-composite-pkg
+      ├── [deploy.yaml]  Deployment YOURSPACE/hello-composite
+      └── Pkg: hello-subpkg
+          ├── [Kptfile]  Kptfile hello-subpkg
+          ├── [deploy.yaml]  Deployment YOURSPACE/hello-sub
+          ├── hello-dir
+          │   └── [configmap.yaml]  ConfigMap YOURSPACE/hello-cm
+          └── Pkg: hello-nestedpkg
+              ├── [Kptfile]  Kptfile hello-nestedpkg
+              └── [deploy.yaml]  Deployment YOURSPACE/hello-nested
+  
+  The fetched package contains setter parameters which can be used to set configuration
+  values from the commandline, list them
+  
+  $ kpt cfg list-setters hello-composite-pkg/
+  
+      hello-composite-pkg/
+               NAME                VALUE        SET BY   DESCRIPTION   COUNT   REQUIRED
+        gcloud.core.project   YOUR_PROJECT_ID                          1       No
+        image                 helloworld-gke                           1       No
+        namespace             YOURSPACE                                1       Yes
+        tag                   0.1.0                                    1       No
+      
+      hello-composite-pkg/hello-subpkg/
+               NAME                VALUE        SET BY   DESCRIPTION   COUNT   REQUIRED
+        gcloud.core.project   YOUR_PROJECT_ID                          1       No
+        image                 helloworld-gke                           1       No
+        namespace             YOURSPACE                                2       Yes
+        tag                   0.1.0                                    1       No
+      
+      hello-composite-pkg/hello-subpkg/hello-nestedpkg/
+               NAME                VALUE        SET BY   DESCRIPTION   COUNT   REQUIRED
+        gcloud.core.project   YOUR_PROJECT_ID                          1       No
+        image                 helloworld-gke                           1       No
+        namespace             YOURSPACE                                1       Yes
+        tag                   0.1.0                                    1       No
+        
+   If you have `gcloud` set up on your local, you can observe that the value of the setter
+   `gcloud.core.project` is set automatically when the package is fetched.  
+   `gcloud` config setters are automatically set deriving the values from the output of
+   `gcloud config list` command, when the package is fetched using [kpt pkg get].
+  
+  ### Set the setter parameters
```

Comment:
We need a better sentence than "set the setter parameters":

1. "Provide parameter values"
2. "Provide setter values"

Reply:
Done.